example/providers.tf

```diff
@@ -4,9 +4,28 @@ data "vault_generic_secret" "nomad_secret_id" {
   path  = "nomad/creds/write"
 }
 
+data "vault_generic_secret" "minio_secrets" {
+  path  = "secret/minio"
+}
+
```

Comment:
I would recommend leave this block in main.tf , due to this block lies in minio context